sos/report/plugins/networkmanager.py

```diff
@@ -110,16 +116,18 @@ def test_nm_status(version=1):
             "nmcli dev show .*": "nmcli_dev_show_sos"
         })
 
+    def postproc_system_connections(self, sc_path):
```

Comment:
curious question, the fact that we're doing the one thing, do we need a new function?

Reply:
Probably now is not needed anymore. I'll fix this and resend.